src/components/tweet-card/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect, useState } from "react";
+import tweet from "../../__mock__/tweet.json";
+
+import {
+  Card,
+  AvatarContainer,
+  Tweet,
+  TweetContainer,
+  UserInfo,
+  InfoConatiner,
+  CreatedAt,
+  TextContainer,
+  FeedbackContainer,
+  Feedback,
+  Followers,
+  FollowersContainer,
+} from "./style";
+import {
+  HeartIcon,
+  MoreIcon,
+  ReplyIcon,
+  RetweetIcon,
+  ShareIcon,
+} from "../../icons/styled-icons";
+import UserAvatar from "../common/avatar";
+import Title from "../common/Title";
+import UserName from "../common/user-name";
+import moment from "moment";
+
+const TweetCard: React.FC = () => {
```

Comment:
expect prop for Tweet object instead of mock data, mock data was used to get the object info only. Once a component has been created, it should expect a prop.

Reply:
ok